fastlane/lib/fastlane/actions/latest_testflight_build_number.rb

```diff
@@ -75,8 +76,12 @@ def self.available_options
                                        env_name: "INTITIAL_BUILD_NUMBER",
                                        description: "sets the build number to given value if no build is in current train",
                                        optional: true,
-                                       is_string: false)
-
+                                       is_string: false),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                       env_name: "FASTLANE_TEAM_ID",
+                                       description: "optional: Your team ID",
```

Comment:
Do we need "optional" in the description? We don't do this for other ConfigItems.